doc/sources/non-scikit-algorithms.rst

```diff
@@ -0,0 +1,38 @@
+.. ******************************************************************************
+.. * Copyright 2024 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+Non-Scikit Algorithms
+=====================
+Algorithms not presented in the original Scikit-learn are described here. All algorithms are 
+available for both CPU and GPU (including distributed mode)
+
+BasicStatistics
+---------------
+.. autoclass:: sklearnex.basic_statistics.BasicStatistics
+
+
+IncrementalEmpiricalCovariance
+------------------------------
+.. autoclass:: sklearnex.covariance.IncrementalEmpiricalCovariance
+
+
+IncrementalBasicStatistics
+--------------------------
+.. autoclass:: sklearnex.basic_statistics.IncrementalBasicStatistics
+
+IncrementalLinearRegression
+---------------------------
+.. autoclass:: sklearnex.linear_model.IncrementalLinearRegression
```

Comment:
```suggestion

IncrementalBasicStatistics
--------------------------
.. autoclass:: sklearnex.basic_statistics.IncrementalBasicStatistics

IncrementalEmpiricalCovariance
------------------------------
.. autoclass:: sklearnex.covariance.IncrementalEmpiricalCovariance

IncrementalLinearRegression
---------------------------
.. autoclass:: sklearnex.linear_model.IncrementalLinearRegression
```

Reply:
Do not want bother more with this suggestion, please follow alphabetic order here please.
* IncrementalBasicStatistics
* IncrementalBasicStatistics
* IncrementalEmpiricalCovariance
* IncrementalLinearRegression